neqo-http3/src/connection_client.rs

```diff
@@ -99,6 +99,7 @@ impl Http3Client {
         http3_parameters: &Http3Parameters,
         now: Instant,
     ) -> Res<Self> {
+        assert!(conn_params.get_max_datagram_frame_size() == 0);
```

Comment:
This is only temporarily. I have added it because if someone accidentally turn it on the code may panic if it receive a Datagram event later. This is just to make code panic non start not later.
I will add a note.

Reply:
I have change my mind: I will remove this assertion and just ignore Datagram* events.